bundles/org.openhab.binding.nikohomecontrol/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -311,9 +443,77 @@
 		<label>@text/channelPowerLabel</label>
 		<description>@text/channelPowerDescription</description>
 		<category>Number</category>
-		<state readOnly="true" pattern="%.0f %unit%"/>
+		<state readOnly="true"/>
```

Comment:
Why removing the default pattern ? This is a good practice to define one.

Reply:
Added back in.